### ISSUE TITLE
Allocation of Ministries

### ISSUE THREAD

**Issue by TRManderson** [lsenjov/politico#11]
The only eminently relevant ministry position as of right now is Financial Minister, but more will become relevant soon. We must decide, not only on ministries, but on ministers.

**Comment by Guthers** [comment]
I have a few minister-ships I would like to suggest
- Foreign
  Includes trade, emigration, immigration policies
- Environmental
  The ecosystems of our country, our agriculture and our sustainable resource management
- Infrastructure
  Making our country bigger and better
- Military
  Protecting our country from threats from within and from the outside
- Welfare
  Making sure the citizens are well cared for, happy, employed and healthy

**Comment by rslabbert** [comment]
I think a few roles should be combined for the immediate future, due to lack of active ruling participants.
- The finance minister can probably assume the role of the trade/foreign relations minister.
- Infrastructure, Environmental, and Welfare can also be combined.

**Comment by Guthers** [comment]
I disagree with both of the combinations. Maybe environmental and infrastructure, but I don't think the finance minister should have control on  i/emigration and trade is another huge significant issue for us all

**Comment by TRManderson** [comment]
I think the trade/financial combination makes a lot of sense, but including foreign relations perhaps not.

Perhaps, in this volatile time in our nation, we should hand foreign relations to the same minister as military?

**Comment by rslabbert** [comment]
That makes sense to me. A think a smaller number of roles would be beneficial at this point in time.

**Comment by rslabbert** [comment]
Here are my current suggestions:
- Welfare
  - Health
  - Education
  - Culture
- Infrastructure
  - Tourism
  - Transport
- Military
  - Foreign Relations
  - Justice
  - Immigration
- Finance
  - Trade
- Science
  - Agriculture
  - Environment

I'm happy to hear opinions and discussion.

**Comment by TRManderson** [comment]
I think, if we very explicitly make it clear that the military managing justice is a transitional measure, that those are very fair allocations of responsibility to minsters

**Comment by rslabbert** [comment]
Definitely, I think we should include in the amendment a certain time period for which the military is allowed to have access to justice.

**Comment by CameronAavik** [comment]
I would also like to propose a Ministry of Magic. Given that our citizens are capable of magic. We need a ministry which  manages the laws surrounding the use of magic. I have not decided on what sub-ministries this will have though.

Secondly I propose a ministry of intelligence. As any good government, we need to make sure that our citizens are safe. The only way to ensure safety is to be one step ahead of all potential threats/terrors. The ministry of intelligence (MOI), will primarily make decisions about what data is collected, and PR regarding data collection.
